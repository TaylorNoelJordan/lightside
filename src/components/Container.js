import React from 'react';
import Card from './Card'

const Container = () => {
    return (
        <section className='cardContainer'>
            <Card />
        </section>
    )
}

export default Container;