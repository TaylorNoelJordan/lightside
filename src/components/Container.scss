.cardContainer {
    display: flex;
    // display: grid;
    // grid-gap: 25px;
    // margin: 25px;
    // grid-template-columns: repeat(auto-fit, minmax(200px, 1fr)); 
}