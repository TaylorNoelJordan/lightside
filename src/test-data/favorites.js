const favorites = [
  {
    "name": "R5-D4",
    "height": "97",
    "mass": "32",
    "hair_color": "n/a",
    "skin_color": "white, red",
    "eye_color": "red",
    "birth_year": "unknown",
    "gender": "n/a",
    "homeworld": "https://swapi.co/api/planets/1/",
    "created": "2014-12-10T15:57:50.959000Z",
    "isFavorite": true
  },
  {
    "name": "Sand Crawler",
    "model": "Digger Crawler",
    "manufacturer": "Corellia Mining Corporation",
    "cost_in_credits": "150000",
    "length": "36.8",
    "max_atmosphering_speed": "30",
    "crew": "46",
    "passengers": "30",
    "cargo_capacity": "50000",
    "consumables": "2 months",
    "vehicle_class": "wheeled",
    "created": "2014-12-10T15:36:25.724000Z",
    "isFavorite": true
  },
]

export default favorites 