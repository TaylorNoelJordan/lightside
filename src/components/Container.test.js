import React from 'react';
import Container from '../components/Container';
import { shallow } from 'enzyme';
import favorites from '../test-data/favorites'
import peopleData1 from '../test-data/people-data-pg1'

describe('Container', () => {
    let wrapper;

    beforeEach(() => {
        wrapper = shallow(
        <Container addFavorite={jest.fn()}
            removeFavorite={jest.fn()}
            favorites={favorites}
            data={peopleData1}
            type={'people'}
            />)
    });

    it('should render card components', () => {
        expect(wrapper).toMatchSnapshot();
    })

})