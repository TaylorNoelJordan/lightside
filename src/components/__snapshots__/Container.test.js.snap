// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container should render card components 1`] = `ShallowWrapper {}`;
